packages/expo-updates/scripts/createManifest.js

```diff
@@ -1,3 +1,5 @@
+const filterPlatformAssetScales = require('@react-native-community/cli/build/commands/bundle/filterPlatformAssetScales')
```

Comment:
I agree with Brent that this coupling is fragile. Looking at https://github.com/react-native-community/cli/blob/66f5a51dccc66a12a4ba9ff9901c468a99148ab1/packages/cli/src/commands/bundle/filterPlatformAssetScales.ts it'd be easy to vendor, but that would likely increase the coupling; our vendored logic would need to stay in sync with whatever RN CLI's logic is, and there'd be even less of an indication that the two pieces of code need to be synced. The real answer I think is to make this part of RN CLI's public API eventually.

Reply:
It seems it breaks `expo eject`

```
Error: Cannot find module '@react-native-community/cli/build/commands/bundle/filterPlatformAssetScales'
Require stack:
- /Users/chrmod/Projects/game/node_modules/expo-updates/scripts/createManifest.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1042:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/Users/chrmod/Projects/game/node_modules/expo-updates/scripts/createManifest.js:1:35)
    at Module._compile (internal/modules/cjs/loader.js:1156:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/chrmod/Projects/game/node_modules/expo-updates/scripts/createManifest.js'
  ]
}
Command PhaseScriptExecution failed with a nonzero exit code
```